regolith/schemas.py

```diff
@@ -1526,7 +1530,12 @@
                          'initial project plan',
             'audience': ['lead', 'pi', 'group_members'],
             'status': 'proposed',
-            'type': 'meeting'
+            'type': 'meeting',
+            'progress':[{
+              - 'text': 'The samples have been synthesized and placed in the sample cupboard. '
```

Comment:
I was finally able to check the error message from Travis and it is here.  You are mixing json and yaml syntax here.  You need to properly format the dict.  line-breaks aside, what you have is `{- "text": "the samples...." - "figure": "<token..."}`.  A properly composed dict would not have the dashes, and would have commas between the items.

Reply:
I changed it, but there's still an issue somewhere. I'll keep staring at it to see if I can find it and fix it.

Reply:
I changed the exemplar, but I think there is still an issue. I'll keep staring at it.